binderhub/templates/page.html

```diff
@@ -32,7 +32,7 @@
   {% block footer %}
   <div class="container">
     <div class="row text-center">
-      <h3>questions?<br />join the <a href="https://gitter.im/jupyterhub/binder">chat</a>, read the <a href="https://mybinder.readthedocs.io/en/latest/">docs</a>, see the <a href="https://github.com/jupyterhub/binderhub">code</a></h3>
+      <h3>questions?<br />join the <a href="https://discourse.jupyter.org">discussion</a>, read the <a href="https://mybinder.readthedocs.io/en/latest/">docs</a>, see the <a href="https://github.com/jupyterhub/binderhub">code</a></h3>
```

Comment:
maybe this should link directly to the binder sub-channel? https://discourse.jupyter.org/c/binder

Reply:
```suggestion
      <h3>questions?<br />join the <a href="https://discourse.jupyter.org/c/binder">discussion</a>, read the <a href="https://mybinder.readthedocs.io/en/latest/">docs</a>, see the <a href="https://github.com/jupyterhub/binderhub">code</a></h3>
```